tutorial/sample-problems/README.md

```diff
@@ -0,0 +1,97 @@
+not exist in the OPL, so were generated to make runnable PG problems.
+
+## Metadata for a Sample Problem
+
+All metadata and documentation are embedded in a problem within comments that
+start with `#:%` or `#:`.  Metadata for a problem can appear anywhere but
+the standard location is just above the `DOCUMENT();` command.  The following
+metadata is currently supported.
+
+- **name**: the name of the problem as will appear in the documentation listings.
+- **type**: the type of the problem can be one or more of the following
+  * _sample_: a sample problem that would appear in some course.
+  * _technique_: a problem from the _Problem Techniques_ on the wiki which may
+  demonstrate something of interest, but may not be relevant as a sample problem.
+  * _snippet_: part of a problem that is useful if embedded in another problem.
+
+- **subject**: A subject area that a sample problem belongs to.  This may be more
+that one.
+- **see_also**: A listing of other problems (by the filename) that will appear
+in the documentation as a reference.
+- **category**: A listing of any categories that the problem would belong to.
+
+In many of these, if more than one is desired, place the list in brackets [ ].
+Also, both the singular and plural forms of the type, subject, category are
+supported.
+
+Here's an example:
+```perl
+#:% name = Surface Graph in Cylindrical Coordinates
+#:% type = Sample
+#:% subject = [Vector Calculus]
+#:% see_also = [SpaceCurveGraph.pg, SurfaceGraph.pg]
+#:% categories = [graph]
+```
+
+## Structure of the code documenation
+
+The comments that are associated with each file in on the WeBWorK
+wiki have been embedded as formatted
+comments in the PGML files. The format for these have the form
+
+```perl
+#:% section = preamble
+#: This is the documentation comments for the preamble
+DOCUMENT();
+
+loadMacros('PGstandard.pl', 'PGML.pl', 'PGcourse.pl');
+
+#:% setup
+#: We use `do { $b = random(2,9,1); } until ( $b != $a );` to generate distinct
+#: random numbers.
+$a = non_zero_random(-9, 9);
+do { $b = random(2,9,1); } until $b != $a;
+
+$answer1 = Compute("$a");
+$answer2 = Compute("($a x^($b) + $b) / x")->reduce();
+```
+
+where the beginning of a documentation block starts with `#:%` followed by a section name, then any options in the
+pattern `opt=value` or `opt:value`. The default names are `preamble`, `setup`, `statement`, `answer`, and `solution`.
+Note that at this point no actual options are honored.
+
+All documentation lines that start with `#:` and are formatted in markdown.
+
+All lines following the documentation lines are considered code until the next `#:%` line is encountered.
+
+## Generate the documentation
+
+The documentation is generated with the `parse-prob-doc.pl` script in the `bin`
+directory of pg. There are the following options (and many are required):
+
+- `problem_dir` or `d`:  The directory where the sample problems are.  This is
+`PG_ROOT/doc/sample-problems`, but required to add.
```

Comment:
This also needs to be updated for the new location.

Reply:
Also, this is no longer required.  I didn't notice this, or I would have changed this before.